src/main/java/games/strategy/engine/framework/ArgParser.java

```diff
@@ -120,10 +120,44 @@ private boolean setSystemPropertyOrClientSetting(
   }
 
   private static boolean handleGameSetting(final String key, final String value) {
-    if (GameRunner.MAP_FOLDER.equals(key)) {
+    if (CliProperties.MAP_FOLDER.equals(key)) {
       ClientSetting.MAP_FOLDER_OVERRIDE.save(value);
       return true;
     }
     return false;
   }
+
+
+  /**
+   * An interface for all CLI related constants.
```

Comment:
Please update this Javadoc, as it's no longer an interface.  You could just say something like "A collection of all CLI-related property names" without having to refer to the type as being a class or an interface.